build.sbt

```diff
@@ -409,26 +421,34 @@ lazy val protobuf: Project = project
     test % "test->test"
   )
 
+val unpackMetadata = taskKey[Seq[File]]("Unpack tensorflow metadata proto files.")
+
 lazy val tensorflow: Project = project
   .in(file("tensorflow"))
   .settings(
     commonSettings,
+    protobufSettings,
     moduleName := "magnolify-tensorflow",
     description := "Magnolia add-on for TensorFlow",
-    Compile / sourceDirectories := (Compile / sourceDirectories).value
-      .filterNot(_.getPath.endsWith("/src_managed/main")),
-    Compile / managedSourceDirectories := (Compile / managedSourceDirectories).value
-      .filterNot(_.getPath.endsWith("/src_managed/main")),
     libraryDependencies ++= Seq(
+      "com.google.protobuf" % "protobuf-java" % protobufVersion % ProtobufConfig,
       "org.tensorflow" % "tensorflow-core-api" % tensorflowVersion % Provided
     ),
-    // Protobuf plugin adds protobuf-java to Compile scope automatically; we want it to remain Provided
-    libraryDependencies := libraryDependencies.value.map { l =>
-      (l.organization, l.name) match {
-        case ("com.google.protobuf", "protobuf-java") =>
-          l.withConfigurations(Some("provided,protobuf"))
-        case _ => l
-      }
+    // tensorflow metadata protos are not packaged into a jar. Manually extract them as external
+    unpackMetadata := {
+      IO.unzipURL(
+        new URL(
+          s"https://github.com/tensorflow/metadata/archive/refs/tags/v$tensorflowMetadataVersion.zip"
```

Comment:
this task is a great idea! 👍 

CCing @martinbomio on this change as he's contributed to magnolify-tensorflow :)

Reply:
would it be possible to inspect the proto files locally?
It is also important to note that this zip contains a few more proto messages than the ones strictly needed

Reply:
All files will be fetched in `tensorflow/target/protobuf_external_src/tensorflow_metadata/proto/v0`.
The zip indeed contains more files than required but they are'nt package anyway.
I'd just keep them to consider the zip as a lib, where it is common to not used every single class inside it